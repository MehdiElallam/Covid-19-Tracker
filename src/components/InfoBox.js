import React from 'react'
import { Col } from 'react-bootstrap'
import styled from 'styled-components';


export default function InfoBox({title, number}) {
    return (
        <Card className='card'>
            <H5>{title}</H5>
            <Small>{number}</Small>
        </Card>
    )
}

const Card = styled.div`
    background: #f9f9f9;
    color: #484848;
    padding: 15px 25px;
`;

const H5 = styled.h5`
    font-size : 14px
`;

const Small = styled.small`
    border-bottom : 1px dotted gray;
    width: 80px;
`;