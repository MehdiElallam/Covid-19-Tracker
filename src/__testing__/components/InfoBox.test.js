import React from 'react';
import { shallow } from 'enzyme';
import InfoBox from '../../components/InfoBox';


describe("Rendring InfoBox component", () => {
    
    it('InfoBox should render correctly with no props', () => {
        
        const component = shallow(<InfoBox/>);
        expect(component).toMatchSnapshot();
    });
    
    
    it('InfoBox should render correctly with props', () => {
        
        const component = shallow(<InfoBox title="Cases" number={110} />);
        expect(component).toMatchSnapshot();
    });

});