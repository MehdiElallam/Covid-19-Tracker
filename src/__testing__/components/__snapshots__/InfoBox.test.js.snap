// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`InfoBox should render correctly with given props 1`] = `ShallowWrapper {}`;

exports[`InfoBox should render correctly with no props 1`] = `ShallowWrapper {}`;

exports[`Rendring InfoBox component InfoBox should render correctly with given props 1`] = `ShallowWrapper {}`;

exports[`Rendring InfoBox component InfoBox should render correctly with no props 1`] = `ShallowWrapper {}`;

exports[`Rendring InfoBox component InfoBox should render correctly with props 1`] = `ShallowWrapper {}`;
